pkg/fanal/analyzer/language/julia/pkg/pkg_test.go

```diff
@@ -0,0 +1,250 @@
+									"cf7118a7-6976-5b1a-9a39-7adc72f591a4",
+								},
+							},
+							{
+								ID:        "aea7be01-6a6a-4083-8856-8a6e6704d82a",
+								Name:      "PrecompileTools",
+								Version:   "1.1.1",
+								Indirect:  true,
+								Locations: []types.Location{{StartLine: 26, EndLine: 30}},
+								DependsOn: []string{"21216c6a-2e73-6563-6e65-726566657250"},
+							},
+							{
+								ID:        "21216c6a-2e73-6563-6e65-726566657250",
+								Name:      "Preferences",
+								Version:   "1.4.0",
+								Indirect:  true,
+								Locations: []types.Location{{StartLine: 32, EndLine: 36}},
+								DependsOn: []string{"fa267f1f-6049-4f14-aa54-33bafae1ed76"},
+							},
+							{
+								ID:        "de0858da-6303-5e67-8744-51eddeeeb8d7",
+								Name:      "Printf",
+								Version:   "1.9.0",
+								Indirect:  true,
+								Locations: []types.Location{{StartLine: 38, EndLine: 40}},
+								DependsOn: []string{"4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"},
+							},
+							{
+								ID:        "9a3f8284-a2c9-5f02-9a11-845980a1fd5c",
+								Name:      "Random",
+								Version:   "1.9.0",
+								Indirect:  true,
+								Locations: []types.Location{{StartLine: 42, EndLine: 44}},
+								DependsOn: []string{"9e88b42a-f829-5b0c-bbe9-9e923198166b", "ea8e919c-243c-51af-8825-aaa63cd721ce"},
+							},
+							{
+								ID:        "ea8e919c-243c-51af-8825-aaa63cd721ce",
+								Name:      "SHA",
+								Version:   "0.7.0",
+								Indirect:  true,
+								Locations: []types.Location{{StartLine: 46, EndLine: 48}},
+							},
+							{
+								ID:        "9e88b42a-f829-5b0c-bbe9-9e923198166b",
+								Name:      "Serialization",
+								Version:   "1.9.0",
+								Indirect:  true,
+								Locations: []types.Location{{StartLine: 50, EndLine: 51}},
+							},
+							{
+								ID:        "fa267f1f-6049-4f14-aa54-33bafae1ed76",
+								Name:      "TOML",
+								Version:   "1.0.3",
+								Indirect:  true,
+								Locations: []types.Location{{StartLine: 53, EndLine: 56}},
+								DependsOn: []string{"ade2ca70-3891-5945-98fb-dc099432e06a"},
+							},
+							{
+								ID:        "cf7118a7-6976-5b1a-9a39-7adc72f591a4",
+								Name:      "UUIDs",
+								Version:   "1.9.0",
+								Indirect:  true,
+								Locations: []types.Location{{StartLine: 58, EndLine: 60}},
+								DependsOn: []string{"9a3f8284-a2c9-5f02-9a11-845980a1fd5c", "ea8e919c-243c-51af-8825-aaa63cd721ce"},
+							},
+							{
+								ID:        "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5",
+								Name:      "Unicode",
+								Version:   "1.9.0",
+								Indirect:  true,
+								Locations: []types.Location{{StartLine: 62, EndLine: 63}},
+							},
+						},
+					},
+				},
+			},
+		},
+		{
+			// If there are no libs, `toApplication` makes the app nil. Not sure why but this is the expected behavior.
+			name: "no_deps_v1.6",
+			dir:  "testdata/no_deps_v1.6",
+			want: &analyzer.AnalysisResult{
+				Applications: nil,
+			},
```

Comment:
As you correct wrote - this is expected behavior. I think we can remove this comment.
```suggestion
			name: "no_deps_v1.6",
			dir:  "testdata/no_deps_v1.6",
			want: &analyzer.AnalysisResult{},
```